IBPSA/Resources/Scripts/BuildingsPy/conf.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
What is this script for? To exclude specific tests from regression? Or is it more an example how to use this new fancy stuff?